### PR TITLE
:memo: bennaaym/docs: update setup documentation

### LINKED ISSUE #516: Issues setting up the project locally
Hello, I'm trying to set up the project locally on my machine however I'm getting the following issues:

- When trying to run `make init`
```
bash ./compose/make_backend_env.sh ./.env
process_begin: CreateProcess(NULL, bash ./compose/make_backend_env.sh ./.env, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:154: .env] Error 2
```

- When trying to run `docker-compose up`:
```
WARNING: The POSTGRES_PASSWORD variable is not set. Defaulting to a blank string.
ERROR: Couldn't find env file: C:\XXX\XXX\Desktop\zubhub\zubhub_backend\.env
```

### Environment:
- Windows 10
- GNU Make 4.3
- Docker Compose version 1.29.2
- Docker version 20.10.14

### PR DESCRIPTION
**Pull request by bennaaym**
## Summary

This PR update the project set-up documentation. This PR solves the issues I faced when I tried to set up the project locally on my machine.

Closes #516

## Changes

- README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,15 +173,21 @@ Visit http://localhost:8000 on your browser to access the API documentation.
 
 ## Setup Frontend
 
-- Create a file named **.env** in the frontend root folder (same directory with package.json)
-- Copy the content of **.env.example** and paste into the new .env file.
 - On your terminal/command line, navigate to **./zubhub/zubhub_frontend/zubhub** directory
 
+- Create a file named **.env** in the frontend root folder (same directory with package.json), then copy-past the following lines
```
Comment (tuxology) [inline README.md]: Typo: `copy-paste` 

Maybe we can make this sentence as,
```
Create a file named **.env** in the frontend root folder (same directory with package.json), with the following content
```

File: README.md [hunk 2]
```diff
@@ -163,15 +173,21 @@ Visit http://localhost:8000 on your browser to access the API documentation.
...
+REACT_APP_NODE_ENV=developement
+REACT_APP_BACKEND_DEVELOPMENT_URL=http://127.0.0.1:8000
+```
+
 ### **Using npm**
 
 This is advisable for a better development experience.
 
-- Run `npm install` to install the dependencies.
+- Run `npm install --legacy-peer-deps` to install the dependencies.
```
Comment (tuxology) [inline README.md:190]: Can you elaborate why we need this dependency?

**Review by tuxology — CHANGES_REQUESTED** [review]
Thanks for the PR. I've left some minor comments

File: README.md (hunk 2 above)
Comment (bennaaym) [reply README.md:190]: The **--legacy-peer-deps** flag was introduced with v7 as a way to bypass peerDependency auto-installation.  Due to the large number of modules that haven't specifically added React v17 as a peerDependency, it's now commonplace to encounter the unable to resolve dependency tree error when running npm installs within a v17 React application. So adding **--legacy-peer-deps** flag will solve the issue.

File: README.md (hunk 1 above)
Comment (bennaaym) [reply README.md]: Done

**Review by tuxology — APPROVED** [review]

**Pull request merged**